### PR TITLE
Make buildx optional with a flag

### PR DESCRIPTION
**Pull request by asosnovsky**
For some reasons when using builx my addon fails to build `armhf` images with base `python:3.10-slim-buster` images.

I get this error:

```
#2 [internal] load .dockerignore
#2 transferring context: 211B done
#2 DONE 0.0s

#3 [internal] load metadata for docker.io/library/python:3.10-slim-buster
#3 ...

#4 [auth] library/python:pull token for registry-1.docker.io
#4 DONE 0.0s

#3 [internal] load metadata for docker.io/library/python:3.10-slim-buster
#3 ERROR: no match for platform in manifest sha256:f17c94905c9cd56dce9ef6ce63229045a75f395f7b5b68eb69ef617079c51848: not found
------
 > [internal] load metadata for docker.io/library/python:3.10-slim-buster:
------
error: failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to create LLB definition: no match for platform in manifest sha256:f17c94905c9cd56dce9ef6ce63229045a75f395f7b5b68eb69ef6[170](https://github.com/asosnovsky/Shortumation/runs/8121900517?check_suite_focus=true#step:5:179)79c51848: not found
Error: Process completed with exit code 1.
```

So after some googling I came across https://github.com/docker/buildx/issues/476 which coupled with the fact that I have been experiencing this issue ever since the last update believe that buildx is the problem.

Please merge this PR so that I can disable it where needed.

Thank you!

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @asosnovsky, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;builder#154).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: builder.sh [hunk 1]
```diff
@@ -19,6 +19,7 @@ DOCKER_PASSWORD=
 DOCKER_LOCAL=false
 SELF_CACHE=false
 CUSTOM_CACHE_TAG=
+DISABLE_BUILDX=true
```
Comment (pvizeli) [inline builder.sh:22]:
```suggestion
DISABLE_BUILDX=false
```

File: builder.sh [hunk 2]
```diff
@@ -300,7 +309,7 @@ function run_build() {
 
     # Build image
     bashio::log.info "Run build for ${repository}/${image}:${version} with platform ${docker_platform}"
-    docker buildx build --pull --tag "${repository}/${image}:${version}" \
+    docker ${build_cmd} --pull --tag "${repository}/${image}:${version}" \
```
Comment (agners) [inline builder.sh:312]:
```suggestion
    docker "${build_cmd}" --pull --tag "${repository}/${image}:${version}" \
```

**Comment by pvizeli** [comment]
make that still sense, also since it's now the default on docker itself? Maybe it got fix with new docker version?

**Comment by agners** [comment]
Good question indeed.

@asosnovsky is this still a problem for you?

**Pull request closed without merging**